modules/installation-ibm-cloud-config-yaml.adoc

```diff
@@ -203,6 +207,101 @@ For production {product-title} clusters on which you want to perform installatio
 ====
 endif::vpc[]
 
+ifdef::private[]
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform:
+    ibm-cloud: {}
+  replicas: 3
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform:
+    ibmcloud: {}
+  replicas: 3
+metadata:
+  name: test-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <5>
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
```

Comment:
machineNetwork cidr need cover the cidrs of the subnets in the vpc.

Reply:
Done. I noticed that we do not call out this fact in the documentation for installing to an existing VPC. Should I update that sample as well?

[1] https://50561--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-vpc.html#installation-ibm-cloud-config-yaml_installing-ibm-cloud-vpc

Reply:
install private cluster based on install on an existing VPC, baseDomain is different based on publish value.
the private cluster fail to prompt the baseDomain value.